tests/tutor/test_tutor_teacher.py

```diff
@@ -0,0 +1,1473 @@
+    # AND: Has at least two students
+
+    # WHEN: Click on a course
+    # AND: Click on the ""Student Scores"" button
+    # AND: Click on ""Name and Student ID"" on the table
+    # AND: Click on ""Name and Student ID"" on the table again
+
+    # THEN: The students are sorted alphabetically by last name.
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_sort_score_by_assignment_completion(tutor_base_url, selenium,
+                                             teacher):
+    """Sort student by completion of an assignment in the score page."""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+    # AND: Has at least two students
+    # AND: Students has somewhat finished an assignment
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the "Student Scores" button
+    # AND: Click "Progress"
+
+    # THEN: Students are sorted by completion of selected assignment.
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_sort_score_by_assignment_score(tutor_base_url, selenium, teacher):
+    """Sort student by scores of an assignment in the score page."""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+    # AND: Has at least two students
+    # AND: Has a graded assignment
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the ""Student Scores"" button
+    # AND: Click ""Score""
+
+    # THEN: Students are sorted by completion of selected assignment.
+
+
+@expected_failure
+@test_case('')
+@tutor
+def test_edit_score_weights(tutor_base_url, selenium, teacher):
+    """Edit score weights as teacher."""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the "Student Scores" button
+    # AND: Click ""set weights"" on the page
+
+    # THEN: A window for edit weights should pop up
+    # AND: user should be able to change the weights from this window
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_cancel_edit_score_weights(tutor_base_url, selenium, teacher):
+    """Cancel editing score weights as teacher."""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the "Student Scores" button
+    # AND: Click "set weights" on the page
+    # AND. click "set weights" on the page
+    # AND.  change some weights in the popup window
+    # AND. click cancel button
+
+    # THEN: User is taken back to the scores page, nothing should change
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_click_see_why_in_score_weights(tutor_base_url, selenium, teacher):
+    """Go to "see why" blog page from score weights as teacher"""
```

Comment:
Two things here: since it's a doctext, use single quotes within to statement and the sentence should end with a period `.`